Example/OriginateRemoteCollection-Example/MyDataSource.h

```diff
@@ -1,17 +1,17 @@
 //
 //  MyDataSource.h
-//  OriginateAsyncCollection-Example
+//  OriginateRemoteCollection-Example
 //
 //  Created by Philip Kluz on 10/1/15.
 //  Copyright © 2015 originate.com. All rights reserved.
 //
 
 @import UIKit;
-@import OriginateAsyncCollection;
+@import OriginateRemoteCollection;
 @import DZNEmptyDataSet;
 
-@interface MyDataSource <__covariant T> : OriginateAsyncCollection <UITableViewDataSource,
-                                                                    UITableViewDelegate>
+@interface MyDataSource<__covariant T> : OriginateMutableRemoteCollection <UITableViewDataSource,
```

Comment:
Agree and agree. Will keep `My` prefix for consistency though -> `MyDateDataSource`.